R/profvis.R

```diff
@@ -140,7 +143,13 @@ print.profvis <- function(x, ..., width = NULL, height = NULL,
   if (viewer) {
     getS3method("print", "htmlwidget")(x)
   } else {
-    NextMethod()
+    f <- getOption("profvis.print")
+    if (is.function(f)) {
+      f(x)
```

Comment:
The `width`, `height`, and `split` arguments actually modify `x` before it's passed along to the next function, so there's no need to pass those along. Similarly, the `viewer` arg is used in this function before the next one is called.

However, I do think it makes sense to pass along the `...`, as in `f(x, ...)`, in case users want to use other arguments. I'll make that change to the `getS3method("print", "htmlwidget")(x)` line in master.